### PR TITLE
Bump concurrently version to fix windows 10 builds

### PR DESCRIPTION
**Pull request by Mellow993993**
Signed-off-by: mellow <stay.mellow.993@gmail.com>

### CODE REVIEW

**Review by Nokel81 — DISMISSED** [review]

File: package.json [hunk 1]
```diff
@@ -329,8 +329,10 @@
     "chart.js": "^2.9.4",
     "circular-dependency-plugin": "^5.2.2",
     "color": "^3.2.1",
-    "concurrently": "^5.3.0",
-    "css-loader": "^5.2.7",
+
+    "concurrently": "^7.0.0",
+    "css-loader": "^6.5.1",
```
Comment (Nokel81) [inline package.json:332]: Do we need to update css-loader too?

Comment (Mellow993993) [reply package.json:332]: No there is no need to upgrade the "ccs-loader" to a newer version, but the concurrently version has to be updated to the version 7.0.0 or higher to make the build process on windows 10.

**Comment by msa0311** [comment]
@Nokel81 can you please help @Mellow993 to get this in. We should ensure windows builds are working fine.

**Comment by Nokel81** [comment]
@Mellow993 are you fine if I push to your fork?

**Comment by Mellow993993** [comment]
> @Mellow993 are you fine if I push to your fork?

yes go ahead

**Review by Nokel81 — DISMISSED** [review]

**Review by msa0311 — DISMISSED** [review]

**Review by Nokel81 — APPROVED** [review]

**Review by msa0311 — APPROVED** [review]

**Pull request merged**